### PR TITLE
Coordinate System

### PR DESCRIPTION
**Pull request by OndraMichal**
# Description

Coordinate System class and functions added. Connected to issue #275.
Coordinate System is only editable through Basic Objects so it makes sence to put it to the same folder.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

# How Has This Been Tested?

- [x] Unit Tests
- [ ] Attached examples

**Test Configuration**:
* RFEM / RSTAB version: grandmaster|6.02.0073.133.ef86f73ef2b
* Python version: 3.11.3

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: RFEM/BasicObjects/coordinate_system.py [hunk 1]
```diff
@@ -0,0 +1,339 @@
+from RFEM.initModel import Model, clearAttributes, deleteEmptyAttributes
```
Comment (dogukankaratas) [inline RFEM/BasicObjects/coordinate_system.py:1]: To be consistent in the naming procedures, I'd suggest to name the file as coordinateSystem.py

File: UnitTests/test_basic_objects.py [hunk 2]
```diff
@@ -946,3 +947,20 @@ def test_get_thickness():
 
     assert thickness['type'] == "TYPE_UNIFORM"
     assert thickness['uniform_thickness'] == 0.02
+
```
Comment (dogukankaratas) [inline UnitTests/test_basic_objects.py:950]: Can we also add tests for other static methods? These would be a reference to our users about the new function.

File: RFEM/BasicObjects/coordinate_system.py (hunk 1 above)
Comment (OndraMichal) [reply RFEM/BasicObjects/coordinate_system.py:1]: Done

File: UnitTests/test_basic_objects.py (hunk 2 above)
Comment (OndraMichal) [reply UnitTests/test_basic_objects.py:950]: fixed

**Review by dogukankaratas — APPROVED** [review]

**Pull request merged**